errors.go

```diff
@@ -18,3 +18,38 @@ func (e errorList) Error() string {
 	}
 	return "multiple errors:\n" + strings.Join(msgs, "\n")
 }
+
+// IgnorableError is an error that has an extra method to tell whether it should
+// be ignored. Ignored errors may be omitted from standard program output.
+type IgnorableError interface {
+	error
+	Ignore() bool
+}
+
+// ignoredError implements IgnorableError, always returning true.
+type ignoredError struct {
+	err error
+}
+
+func (e *ignoredError) Error() string {
+	if e.err == nil {
+		return ""
+	}
+	return e.err.Error()
+}
+
+func (e *ignoredError) Ignore() bool {
```

Comment:
Yeah, I was pondering it. My line of thought was that the error is only ignored after you ignore it :)
IOW `Ignore()` returns whether I may ignore the error or not.

I started sketching this out from the interface definition in https://github.com/feedhenry/fh-system-dump-tool/pull/50/files#diff-dd25e0e0a594c3f3592c1c717039b85eR22. There I say what the method is about.
If we make it `Ignored()`, wouldn't it sound like an error has already been ignored (and now it is "too late to make a decision")?